apps/web/src/config/swell.ts

```diff
@@ -1,5 +1,21 @@
 export const POSITION_MANAGERS_WITH_SWELL_REWARD = [
   {
+    // WETH-swETH
     identifier: '0x2b0Ca7A851aAc56fcACAB3ee85B8Dd96DE799aCa',
   },
+  {
+    // WETH-rswETH
+    identifier: '0x6a38e12CcDe7e35E818b341964fB8AA3BDBb80eE',
+  },
+] as const
+
+export const FARMS_WITH_SWELL_REWARD = [
```

Comment:
maybe we can put them together, what do you think? Since we don't really care about what exactly is the identifier